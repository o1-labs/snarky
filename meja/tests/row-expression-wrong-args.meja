let x : [< `A(bool) | `B | `C > `A] = `A(1);
