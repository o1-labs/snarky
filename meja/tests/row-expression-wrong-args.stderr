File "tests/row-expression-wrong-args.meja", line 1, characters 41-42:
Error: Incompatible types bool and int:
       Cannot unify bool and int
